akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/ActorRefFactoryAdapter.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.internal.adapter
+import akka.actor.typed._
+import akka.annotation.InternalApi
+import akka.{ ConfigurationException, actor => untyped }
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[typed] object ActorRefFactoryAdapter {
```

Comment:
is this replacing the previous ActorContextAdapter, what was the reason for the change (so I know what to look for)?

BTW, wasn't ActorContextAdapter a good name since it seems to be things for the ActorContext and not ActorRefFactory in general (ActorRefFactory can be an ActorContext or an ActorSystem)

Reply:
It was to share the code for ActorContext and ActorSystem for spawning. I shouldn't have moved toUntyped as that is specific to context, fixing now.

I have it on my TODO list to see if we can have one implicit ops class for both context and actor system.